tests/v2/test_waku_store.nim

```diff
@@ -296,29 +296,18 @@ procSuite "Waku Store":
       decodedEmptyIndex.isErr == false
       decodedEmptyIndex.value == emptyIndex
 
-
-  test "PagingDirection Protobuf encod/init test":
-    let
-      pagingDirection = PagingDirection.BACKWARD
-      pb = pagingDirection.encode()
-      decodedPagingDirection = PagingDirection.init(pb.buffer)
-
-    check:
-      # the decodedPagingDirection must be the same as the original pagingDirection
-      decodedPagingDirection.isErr == false
-      decodedPagingDirection.value == pagingDirection
-
   test "PagingInfo Protobuf encod/init test":
     let
       index = computeIndex(WakuMessage(payload: @[byte 1], contentTopic: ContentTopic(1)))
-      pagingInfo = PagingInfo(pageSize: 1, cursor: index, direction: PagingDirection.BACKWARD)
+      pagingInfo = PagingInfo(pageSize: 1, cursor: index, direction: PagingDirection.FORWARD)
```

Comment:
Wondering why this is changed to `FORWARD`, did you find any issue with the `BACKWARD` option?

Reply:
No issues at all. I changed because `PagingDirection.Forward` has a value of `1`, and I wanted to be sure that decoding the direction would return the same and not a default value of `0`. (I don't even know if it's a possible scenario)

Reply:
Oh I see, makes sense!